test/e2e/todo-header-transform-sizes.spec.js

```diff
@@ -0,0 +1,34 @@
+// @reviewer, I have no idea where to put these tests in a way that it would make sense
```

Comment:
```suggestion
```

I would move tests to newly created `transform.spec.js`

Reply:
The other test files are in the walkontable suite, I don't think I could just use the `handsontable` there, would have to rewrite the test. I moved it somewhere else for now, hope it'll be ok being there. https://github.com/handsontable/handsontable/pull/7496/commits/edab6fec22946960f3788357b224a16d23a328cd